scrapers/data18.yml

```diff
@@ -2,34 +2,24 @@ name: data18
 sceneByURL:
   - action: scrapeXPath
     url:
-      - data18.com/content
+      - data18.com/scenes
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     common:
-      $performer: //p[b[text()='Starring:']]/a
-      $studio: //p[b[text()='Site:' or text()='Studio:']]/a
+      $performer: //b[text()='Pornstars / Cast']/../..//a[@class='bold gen']
+      $studio: (//b[text()="Studio"] | //b[text()="Network"])/../a[last()]
     scene:
-      Title: //h1/text()
+      Title: //span/following-sibling::h1/a/text()
       Date:
-        selector: //span[contains(text(),"Release date:") and not(contains(text(),"No date release yet"))]
-        postProcess:
-          - replace:
-              - regex: "^Release date:|,"
-                with:
-              # When day of month is not specified, the scraper defaults to <Month> 1st, <Year>.
-              # Comment the next two lines if you want to skip partial dates instead.
-              - regex: ([A-Za-z]+) (\d{4})
-                with: "$1 1 $2"
-          - parseDate: January 2 2006
+        selector: //time/@datetime
       Details:
-        selector: //p[b[text()='Story:']]
+        selector: //div[b[text()="Story"]]/text() |  //div[b[text()="Story"]]/a/text()
+        concat: ' '
         postProcess:
           - replace:
-              - regex: "^Story:"
-                with:
-              - regex: �
-                with: "'"
+            - regex: "^- "
+              with:
```

Comment:
For the xpath selectors the order you do the union matters.
In this case it is not needed though.
You could go with 
either
```yaml
      Details:
        selector: //div[b[text()="Story"]]//text()
        concat: ' '
        postProcess:
          - replace:
            - regex: "^Story - "
              with:
```
or even simpler since stash gets the inner text

```yaml
      Details:
        selector: //div[b[text()="Story"]]
        postProcess:
          - replace:
            - regex: "^Story - "
              with:
```
not tested with a lot of scenes so might need some adjustment